src/components/OverlayManager.js

```
@@ -0,0 +1,215 @@
+    // Register overlaysRoot with the controller.  The call is idempotent so we
+    // can safely trigger it even if another component has already initialised
+    // the controller.
+    DraggableController.init({ overlaysRoot: this.container });
+  }
+
+  /* -------------------------------------------------------------------- */
+  /* Store handling                                                       */
+  /* -------------------------------------------------------------------- */
+
+  _onStoreChange(newState, prevState) {
+    if (newState.mapping !== (prevState && prevState.mapping)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)